Hw1

wenditer opened the pull request.

wenditer committed:
HW1 - Installing Node.js & Configuring Babel with Nodemon

wenditer committed:
HW1 - Adding Modules

abruzgin commented on src/models/user.js:
```diff
@@ -0,0 +1,7 @@
+  }
+}
+
+export { User }
```
in this case better do `export default User` if you have only 1 thing to export

abruzgin replied:
or even `export default class User { ... }`

abruzgin commented on src/app.js:
```diff
@@ -0,0 +1,8 @@
+console.log(config.name);
+
+import { User, Product } from "./models/models"
+const user = new User();
```
Here you can only call `new User()` without assignment as it's only calls the constructor

wenditer committed:
HW1 - Adding Modules

- code review fixes, export changed to default, removed redundant argument "watch" from npm start script, previous way of exporting modules left in comments just for reference.
